scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/fgboost/VflGBoostAggregator.scala

```diff
@@ -182,59 +238,47 @@ class VflGBoostAggregator extends Aggregator[Table] {
       .putTable("label", toFloatTensor(label))
       .build()
     // Update gradient
-    _evalMap.clear()
-    trainStorage.updateStorage(aggregatedModel)
+    tableStorage.clientData.clear()
+    tableStorage.updateStorage(aggregatedModel)
```

Comment:
`updateStorage` will clear `clientData`, yes?

maybe change `updateStorage` to something like `clearAndUpdateServer`?

Reply:
`updateStorage` would just replace the server data with arg. But surely we can merge clear client data and updateStorage in a `clearAndUpdateServer` method. But I am not sure if there are separate usage of them

Reply:
> `updateStorage` would just replace the server data with arg. But surely we can merge clear client data and updateStorage in a `clearAndUpdateServer` method. But I am not sure if there are separate usage of them

See https://github.com/intel-analytics/BigDL/blob/branch-2.0/scala/ppml/src/main/java/com/intel/analytics/bigdl/ppml/common/Storage.java#L43